var/ramble/repos/builtin/applications/hpl/application.py

```diff
@@ -17,120 +17,180 @@ class Hpl(SpackApplication):
 
     maintainers('douglasjacobsen', 'dodecatheon')
 
-    tags('benchmark-app', 'mini-app', 'benchmark')
+    tags('benchmark-app', 'benchmark', 'linpack')
 
     default_compiler('gcc9', spack_spec='gcc@9.3.0')
 
-    software_spec('impi2018', spack_spec='intel-mpi@2018.4.274')
+    software_spec('impi_2018', spack_spec='intel-mpi@2018.4.274')
 
-    software_spec('hpl',
-                  spack_spec='hpl@2.3 +openmp',
-                  compiler='gcc9')
+    software_spec('hpl', spack_spec='hpl@2.3 +openmp', compiler='gcc9')
 
     required_package('hpl')
 
     executable('execute', 'xhpl', use_mpi=True)
 
     workload('standard', executables=['execute'])
+    workload('calculator', executables=['execute'])
 
-    workload_variable('output_file', default='HPL.out      output file name (if any)',
+    workload_variable('output_file',
+                      default='{:13} output file name (if any)'.format('HPL.out'),
```

Comment:
What is this change for? I see it's fairly pervasive, but it also seems unnecessary. Is there a reason why this is pushed through everywhere?

Reply:
Can we pull this out into a `pad_value` function, so the `'{:13}'` is not repeated in all of the workload variables?

Reply:
Changed and updated.